README.md

```diff
@@ -1,5 +1,97 @@
+## App pages
+
+---
+
+-   **Homepage**
+
+    _This is the main screen of the site where the user will be exposed to the best-selling items, the recommendations of the stylists and will also be able to see the variety of existing categories and choose to view items in a specific category._
+
+---
+
+-   **Registration page**
+
+    _This page accepts new user credentials and creates a new use in the database._  
+    _It is possible for the user to register via his Facebook or Gmail account._  
+    _After registration the user will be redirected to the login page._
+
+    _Please note that the user can choose not to register and still view the site, but if he wants to make a purchase he will have to register first._
+
+---
+
+-   **Login page**
+
+    _This page is responsible for connecting a registered user to the site by checking his details on database._
+
+---
+
+-   **Category page**
+
+    _If the user selected one of the categories that appear on the home page, he will be sent to this page ,where the clothing details classified by the category he has chosen._  
+    _Each item in the list of items will have a picture, a short description (including price)._  
+    _The user can mark favorite items by clicking on a heart icon that will appear in the corner of the item image._
+
+---
+
+-   **Product page**
+
+    _If the user wants to view a particular item in detail - he will be moved to this page when the item is clicked._  
+    _This page will display an extended description of the product, including the option to select a size and view additional images of it._
+    _A cart icon will also appear if the user wants to purchase the product._
+
+---
+
+-   **Favorites page**
+
+    _This page will contain all the items that the user has marked in the heart icon._
+    _Items marked as Favorites by a user who is not registered on the site will be deleted when leaving the site._  
+    _For a registered user the favorite items will be saved on the page._
+
+---
+
+-   **Cart page**
+
+    _This page will contain all the items that the user is interested in purchasing._  
+    _It will be possible to delete or add items to the cart on this page._
+
+---
+
+-   **Purchase page**
+
+    _A user who wishes to proceed with the purchase of the products in the cart will be redirected to this page._
+    _Here he will fill in the credit card information, residential address and more._
+
+---
+
+---
+
+## Navigation menu will contain the following links :
+
+---
+
+-   **User account page :**
+
+    -   _Links to the registration / login / logout page (depending on user status)_
+    -   _Previous purchases_
+    -   _user information_
+    -   _Notificatios_
+    -   _Light / Dark display mode_
+    -   _Ways of contact and general information_
+    -   _Purchase of a gift card_
+
+-   **Favorites page**
+-   **Cart page**
+-   **Search button**
+-   **Homepage**
+
+---
+
+## Technologies:
+
+The app is written by React-TypeScript.
```

Comment:
1. React - UI package from Facebook.
2. Vite - Next Generation Frontend Tooling.
3. Typescript - A super-set of javascript.
4. Styled-components - Visual primitives for the component age. 
5. Storybook - A tool for building UI components.
6. Eslint - Linter for JavaScript code (+ airb&b config).
7. Prettier - An opinionated code formatter.
8. Git - Version control system.
9. Github - Version control sharing platform.

etc...